index.css

```diff
@@ -49,7 +49,7 @@ header h1 sub .community-edition {
 
 main {
 	width: 100%;
-	max-width: var(--max-content-width);
+	max-width: 84em;
 	margin-left: auto;
```

Comment:
This was 70 em and everything could easily fit into it. Why the increased width?

Reply:
I studied the flex of the buttons and made these changes so as to get the desired arrangement.

Reply:
No, these changes broke it even more.